training_project/pages/dashboard_page.py

```diff
@@ -0,0 +1,15 @@
+from selenium.webdriver.common.by import By
+
+from training_project.pages.base_page import BasePage
+
+
+class DashboardPage(BasePage):
+
+    def __init__(self, driver):
+        super().__init__(driver)
+
+    DASHBOARD_PAGE = (By.XPATH, '//*[@id="app"]/div[1]/div[2]/div[2]')
+
+    def dashboard_displayed(self):
+        dashboard = self.find_element(self.DASHBOARD_PAGE)
```

Comment:
I like this check more than check for LoginPage. Better to find some unique element and check if page is displayed by it than by url. But this element is not unique, this locator also present on other pages, so we can't say that it is DshboardPage is displayed